### PR TITLE
[DNM] GCP Deployment

### PR DESCRIPTION
**Pull request by caugner**

### CODE REVIEW

File: app.yaml [hunk 1]
```diff
@@ -8,4 +8,4 @@
 
 runtime: nodejs16
 env_variables:
-  GCLOUD_STORAGE_BUCKET: mdn-bcd-buffer
+  GCLOUD_STORAGE_BUCKET: moz-fx-dev-fiji-mdnplaygound.appspot.com
```
Comment (foolip) [inline app.yaml:11]: Is "playgound" a typo?

**Comment by foolip** [comment]
@caugner let me know if/when you want review of this. Secrets management is something I've never been totally happy with, but I think the simplest way is to put secrets into GitHub Actions and create a `secrets.json` using them before deploy. You could also put secrets somewhere in GCP, but I don't know what the current best practices are.

**Comment by foolip** [comment]
Also note https://github.com/foolip/mdn-bcd-collector/pull/2756. I'm not sure if you're trying to use the same nodejs version for all projects, if so feel free to change it to whatever you like.

**Comment by foolip** [comment]
@caugner do you have an update on this and on the blockers for migrating the collector more generally? cc @Rumyra

**Comment by foolip** [comment]
Ping @caugner for an update :)

**Comment by caugner** [comment]
Closing, as I don't have any capacity to work on this in the near future.

**Pull request closed without merging**